planning/standards-status.md

```diff
@@ -0,0 +1,152 @@
+| POP3                                | [^rfc1939]                            |  &#10004;      |   &#10004;     |                              |         |
+| IMAP 4r1                            | [^rfc3501]                            |  &#10004;      |   &#10004;     |                              |         |
+| IMAP IDLE                           | [^rfc2177]                            |  &#10004;      |   &#10004;     |                              |         |
+| SMTP                                | [^rfc821]                             |  &#10004;      |   &#10004;     |                              |         |
+| NNTP                                | [^rfc977]                             |  &#10004;      |   &#10004;     |                              |         |
+
+[^rfc1939]: https://datatracker.ietf.org/doc/html/rfc1939
+[^rfc3501]: https://datatracker.ietf.org/doc/html/rfc3501
+[^rfc2177]: https://datatracker.ietf.org/doc/html/rfc2177
+[^rfc821]: https://datatracker.ietf.org/doc/html/rfc821
+[^rfc977]: https://datatracker.ietf.org/doc/html/rfc977 
+
+## RSS Feeder
+
+| Spec Name                           | Spec URLs                             | Desktop Status | Android Status | Standards Involvement        | Comment | 
+|:------------------------------------|:-------------------------------------:|:--------------:|:--------------:|:---------------------------- |:------- |
+| RSS 0.9                             | [^rss-0-9-0]                          |  &#10004;      |      N/A       |                              |         |
+| RSS 2.0                             | [^rss-spec]                           |  &#10004;      |      N/A       |                              |         |
+| Atom 0.3                            | [^atom-0-3]                           |  &#10004;      |      N/A       |                              |         |
+| Atom 1.0                            | [^atom-1-0]                           |  &#10004;      |      N/A       |                              |         |
+
+[^rss-0-9-0]: https://www.rssboard.org/rss-0-9-0
+[^rss-spec]: https://www.rssboard.org/rss-specification
+[^atom-0-3]: https://datatracker.ietf.org/doc/html/draft-ietf-atompub-format-00
+[^atom-1-0]: https://datatracker.ietf.org/doc/html/rfc4287
+ 
+## Security and Authentication
+
+| Spec Name                           | Spec URLs                             | Desktop Status | Android Status | Standards Involvement        | Comment |
+|:------------------------------------|:-------------------------------------:|:--------------:|:--------------:|:---------------------------- |:------- |
+| MIME                                | [^rfc2045], [^rfc2046], [^rfc2047]    |  &#10004;      |   &#10004;     |                              | All 3 are supported for both Desktop and Android platforms. |
+| List Management Headers             | [^rfc2369], [^rfc5064]                |  &#10004;      |                |                              |         |
```

Comment:
I would  probably move this to it's own "List management" section and list the headers there.
I'm not sure if it belongs to `Security and Authentication` section.
Just a nit suggestion, not a blocker.